README.md

```diff
@@ -41,17 +41,30 @@ Generally, you'll want to build from the `master` branch, unless you're testing
 
 You should use the latest Java 17 build to run Phantazm.
 
-The proxy and server use the BungeeCord protocol to communicate by default. This requires you to set up a shared secret (string) that the proxy will use to authenticate itself to the server. The server *will refuse to run* with the default secret (an empty string), for security reasons. You'll need to configure your own secret.
+### For development
 
-First, come up with an appropriate secret. You can use a password manager, or a generation tool like `apg` or `pwgen`. If you're just hosting Phantazm locally, with no intention of port-forwarding, you don't have to worry too much about password strength. However, if your server is accessible from the Internet, make sure it's suitably complex. Keep in mind that your secret isn't something you need to actually *remember*, and thus there's no real drawback to making it as long as you want.
+The proxy and server use the BungeeCord protocol to communicate by default. This requires the use of a shared secret (string). This string is used to authenticate the proxy and ensure that no players can connect directly to the underlying server.
```

Comment:
isn't bungeeguard secret not shared

Reply:
it is? how else would the server work lol

Reply:
what does shared mean exactly here?

Reply:
same key on proxy & server (there is no key exchange, so "shared" just means both parties have the key beforehand)

Reply:
after a little bit of further reading I don't believe minestom actually does bungeeguard validation